scico/functional/_norm.py

```diff
@@ -259,3 +261,41 @@ def prox(
         new_length = 0.5 * (new_length + snp.abs(new_length))
 
         return new_length * direction
+
+
+class NuclearNorm(Functional):
+    r"""Nuclear norm.
+
+    Compute the nuclear norm
+
+    .. math::
+      \| X \|_* = \sum_i \sigma_i
+
+    where :math:`\sigma_i` are the singular values of matrix :math:`X`.
+    """
+
+    has_eval = True
+    has_prox = True
+    is_smooth = False
```

Comment:
I thought we did away with this. Or was it just we stopped checking it?

Reply:
This will be taken care of in #89, but the `is_smooth` has to  be defined until that's merged to avoid errors from a base class.